src/pages/index.js

```diff
@@ -3,25 +3,57 @@ import "./index.css";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import PopupWithForm from "../components/PopupWithForm.js";
+import PopupWithConfirm from "../components/PopupWithConfirm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import Section from "../components/Section.js";
```

Comment:
The like button is not clickable, I can't changer the likes state

Reply:
Can you explain? I see when i click the like button, the number is changing so there is some functionality on the like button however Im unsure why the active class is not applying.

Reply:
Note, that `classList.add` accepts a class, not a selector `.classList.add("card__like-button_active")`. The same for class removal